migrations/54-60/removed-backward-incompatibility.md

```diff
@@ -242,3 +242,19 @@ $table = new \Joomla\CMS\Table\Content($db);
 - PR: https://github.com/joomla/joomla-cms/pull/45425
 - File: libraries/src/Application/WebApplication.php
 - Description: The `$item_associations` was added to the `WebApplication` class for improved PHP 8.2 compatibility and is not used at all.
+
+### dispatchEvent proxy functions removed in view and model
+
+- PR: https://github.com/joomla/joomla-cms/pull/45431
+- Files: 
+  - libraries/src/MVC/Model/BaseDatabaseModel.php
+  - libraries/src/MVC/View/AbstractView.php
+- Description: The `dispatchEvent` proxy functions are removed in the `BaseDatabaseModel` and `AbstractView` classes. The `getDispatcher` function therefor requires no a dispatcher injected, which is the default, when the component is booted through the application.
```

Comment:
```suggestion
- Description: The `dispatchEvent` proxy functions are removed in the `BaseDatabaseModel` and `AbstractView` classes. The `getDispatcher` function therefore requires now a dispatcher injected, which is the default when the component is booted through the application.
```
Besides spelling and grammar I assumed that the "no" should be a "now", but that's just my assumption so it might be wrong and you meant something else, e.g. "not".